src/Backups/BackupsWorker.h

```diff
@@ -91,6 +91,9 @@ class BackupsWorker
     std::unique_ptr<ThreadPool> backups_thread_pool;
     std::unique_ptr<ThreadPool> restores_thread_pool;
 
+    std::unique_ptr<ThreadPool> backup_async_executor_pool;
```

Comment:
There can be also `BACKUP ON CLUSTER ASYNC` command which waits `BACKUP ASYNC` to be competed on the nodes of the cluster, so we need extra thread pools for that. I've implemented that in https://github.com/ClickHouse/ClickHouse/pull/55216